skin/src/build.gradle

```diff
@@ -73,6 +74,7 @@ dependencies {
     compile project(':backbone')
     testCompile 'junit:junit:4.12'
     testCompile 'org.robolectric:robolectric:3.0'
+    testCompile 'org.mockito:mockito-core:1.10.19'
```

Comment:
@nrlakin Is this line and the `sourceSets` lines something you added/merged for the Travis-CI stuff?

Not sure where it came from, but seems like it's not necessary to this PR.